### DISCUSSION TITLE
Apache Airflow environment is unable to load UI for the code.

### DISCUSSION THREAD

**Discussion by geetakew** [apache/airflow#20825] [General]
### Apache Airflow version

2.0.2

### What happened

I created airflow environment on AWS by following steps from https://towardsdatascience.com/set-up-an-airflow-environment-on-aws-in-minutes-f934cf10ec54. 

After the environment got available, I clicked on Airflow UI. But I am getting below message:

Something bad has happened.
Please consider letting us know by creating a bug report using GitHub.

Python version: 3.7.10
Airflow version: 2.0.2
Node: redact
-------------------------------------------------------------------------------
Error! Please contact server admin.

### Operating System

AWS Airflow

### Deployment

MWAA

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
There is no way anyone could help you with that level of details. You need to provide all the details of errors and some initial diagnosis if you need help with it so that it's even remotely possible that someone could help.

**Comment by potiuk** [comment]
Converted it to discussion anyway.
